dnf/yum/commands/deplist.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2016 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from dnf.i18n import _
+from dnf.cli.commands.repoquery import RepoQueryCommand
+
+
+class DeplistCommand(RepoQueryCommand):
+    """
+    some comment about command
+    """
+
+    aliases = ('deplist',)
+    summary = _("List a package's dependencies")
```

Comment:
It is grammatically correct. It just lists dependencies of one package and what you suggested lists dependencies of more packages.